README.md

```diff
@@ -26,6 +26,7 @@ In addition to deploying new clusters, this playbook also support the deployment
     - [[Type A] PostgreSQL High-Availability with HAProxy Load Balancing](#type-a-postgresql-high-availability-with-haproxy-load-balancing)
     - [[Type B] PostgreSQL High-Availability only](#type-b-postgresql-high-availability-only)
     - [[Type C] PostgreSQL High-Availability with Consul Service Discovery (DNS)](#type-c-postgresql-high-availability-with-consul-service-discovery-dns)
+    - [[Type D] PostgreSQL High-Availability with HAProxy Load Balancing and Patroni raft](#type-d-postgresql-high-availability-with-haproxy-load-balancing-and-patroni-raft)
```

Comment:
I don't think dcs_type: "patroni_raft" deserves to be described as a separate scheme.

These are all the same type A and type B schemes, just instead of etcd, you can choose  "patroni_raft" in dcs_type

Reply:
consider just adding a comment to the type A schema